### DISCUSSION TITLE
语法错误是什么鬼

### DISCUSSION THREAD

**Discussion by LittleNightSong** [antoor/antSword#357] [Q&A] [answered]
![image](https://github.com/AntSwordProject/antSword/assets/142714722/5a6b113d-95e2-4746-87d5-2275b1bada45)

**Comment by Medicean** [comment]
nodejs 不能用 jscript 来解释执行。
看手册:
https://www.yuque.com/antswordproject/antsword/srruro

**Reply by LittleNightSong** [reply]
OK谢谢;我在下node了。

**Reply by Medicean** [reply, marked as answer]
@SongzqInChina  用 Loader 加载，不要直接用 nodejs。加载器里内置了node

**Reply by LittleNightSong** [reply]
@Medicean 
![image](https://github.com/AntSwordProject/antSword/assets/142714722/d45ce3fa-ce5b-4a70-9014-e1c0cdbf7d1f)
事情越发奇怪了
![00F11FF7](https://github.com/AntSwordProject/antSword/assets/142714722/5981deb1-f70a-4a8c-aa88-04ca9baed689)

**Reply by LittleNightSong** [reply]
没事了。代码被我之前改了点。现在改回来了
